test/basic-querying.test.js

```diff
@@ -432,9 +432,9 @@ describe('basic-querying', function () {
       });
     });
 
-    it('should support boolean "lt" that is satisfied', function (done) {
-      User.find({order: 'seq', where: { vip: { "lt": true }
-      }}, function (err, users) {
+    it('should support boolean "lt" that is satisfied', function(done) {
+      User.find({ order: 'seq', where: { vip: { 'lt': true },
+      }}, function(err, users) {
```

Comment:
Yeah, I know this style is not what we would like to see, but please let's not get too strict about existing legacy code. As I mentioned before, I'd like to keep the amount of changes as small as possible, to make it easier to use  `git blame`.